.circleci/config.templ.yml

```diff
@@ -770,6 +776,9 @@ jobs:
 
   pytest:
     <<: *contrib_job
+    docker:
+      - image: *ddtrace_dev_image
```

Comment:
Why is this needed?

Reply:
this adds the test agent to the job, it originally wasn't being used for pytest